CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,24 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Get Deferred Financial type
+   *
+   * @return array
+   *
+   */
+  public static function getDeferredFinancialType() {
+    $deferredFinancialType = array();
+    $query = "SELECT ce.entity_id, cft.name FROM civicrm_entity_financial_account ce
+INNER JOIN civicrm_option_value cv ON ce.account_relationship = cv.value
```

Comment:
@eileenmcnaughton I agree with issue of joining on int against varchar
https://issues.civicrm.org/jira/browse/CRM-18992